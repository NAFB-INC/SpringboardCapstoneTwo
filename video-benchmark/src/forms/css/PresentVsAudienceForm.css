.PresentVsAudienceForm{
    font-size:65%;
    font-style: initial;
    margin-top:2%;
}

.major-choice-2{
    margin-left:10%;
    margin-right:10%;
    margin-top:5%;
}

.major-choice-btn{
    font-size: 30px;
    font-family: Raleway;
    line-height: 500%;
    padding: .618em 1em;
    border-radius: 10px 10px 12px 12px;
    box-shadow:
      0px 8px 0px 0px #94585a, /* button thickness */
      0px 0 20px rgba(255,255,255,.2) inset, /* inner glow */
      2px 30px 0px rgba(255,255,255,.1) inset, /* sublte reflection */
      5px 15px 30px -10px #000; /* dark shadow underneath */
    border: 2px solid #94585a;
    cursor: pointer;
    background: #A26769;
    color: #ecf0f1;
    text-shadow: 1px 1px 1px #34495e;
    transform: rotateX(5deg);
    margin: 1% 1%;
    width:40%;
    text-align: center;
  }

  .major-choice-btn:focus{
    box-shadow:
      0px 8px 0px 0px #a05f62, /* button thickness */
      0px 0 20px rgba(255,255,255,.2) inset, /* inner glow */
      2px 30px 0px rgba(255,255,255,.1) inset, /* sublte reflection */
      5px 15px 30px -10px #000; /* dark shadow underneath */
    border: 2px solid #a05f62;
    background: #865556;
    color: #d9dedf;
    text-shadow: 1px 1px 1px #34495e;
  }