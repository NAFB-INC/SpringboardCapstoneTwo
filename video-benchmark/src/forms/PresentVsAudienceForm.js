import "./css/PresentVsAudienceForm.css";
import React from "react";

function PresentVsAudienceForm({setPvA}) {

    return (
        <div className="PresentVsAudienceForm">
            Are you the Presenter or the Audience?
            <div className="major-choice-2">
                <button className="major-choice-btn" onClick={()=>setPvA("Presenter")}>I Am The Presenter</button>
                <button className="major-choice-btn" onClick={()=>setPvA("Audience")}>I Am Amongst The Audience</button>
            </div>
        </div>
    );
}

export default PresentVsAudienceForm;