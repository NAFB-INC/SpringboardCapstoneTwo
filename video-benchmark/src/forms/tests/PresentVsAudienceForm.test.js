import React from "react";
import { render } from '@testing-library/react';
import PresentVsAudienceForm from '../PresentVsAudienceForm';

it("renders without crashing", function() {
  render(<PresentVsAudienceForm />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<PresentVsAudienceForm />);
  expect(asFragment()).toMatchSnapshot();
});