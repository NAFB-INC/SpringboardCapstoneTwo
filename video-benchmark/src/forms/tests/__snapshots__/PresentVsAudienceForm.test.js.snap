// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="PresentVsAudienceForm"
  >
    Are you the Presenter or the Audience?
    <div
      class="major-choice-2"
    >
      <button
        class="major-choice-btn"
      >
        I Am The Presenter
      </button>
      <button
        class="major-choice-btn"
      >
        I Am Amongst The Audience
      </button>
    </div>
  </div>
</DocumentFragment>
`;
